pages/ghg/ghg.md

```diff
@@ -0,0 +1,68 @@
+3. **Identify partners along the value chain.** Identify any partners along the value chain that potentially contribute significant GHGs to identify sources, relevant data and calculate emissions.
+4. **Quantify Scope 3 emissions.** It is accepted that Scope 3 emissions may not be accurate due to data availability and reliability. Estimates are acceptable as long as the approach, and methodology is transparent, and that the data used is adequate and reliable.
+
+Inline with this recommended approach, the TCS describes the technology value chain across three categories, identifiying potential sources of technology emissions. The company should carefully consider these categories and sources of emissions that are most relevant to them. Using the [TCS Carbon Estimator tool](/estimator) can help identify which categories and sources may be relatively large and contribute high GHGs. Finally, the TCS provides guidance for quantifying these emissions, with details of appropriate approaches, methodology and sources of reliable data.
+
+The TCS defines three categories that fall within GHG Protocol Scope 3:
+
+{% include categoryLabel.html label="CatU" %}
+
+The upstream value chain; TCS Category U (Upstream), encompasses the embodied carbon of technology including all employee devices, networking equipment and servers. This can make up a significant portion of the organisation's overall GHG emissions. [Product Carbon Footprints](/resources/glossary#product-carbon-footprint-pcf) are generally easily accessible for reporting. However, the upstream impact of purchasing off-the-self and open-source software, is still very difficult to obtain.
+
+{% include categoryLabel.html label="CatC" %}
+
+Category C (Cloud), encompasses the supply of services to the organisation, such as cloud servcies, software-as-a-service (SaaS) and managed services. Notably, it also considers the off-site use of employee devices such as laptops used by employees working from home, for example. For GHG Protocol reporting, high-level, top-down methodologies are suitable for calcualting emissions. Although inaccurate, a cost-based approach for estimating cloud services is a common approach. For off-site employee devices, the [typical energy consumption (TEC)](/resources/glossary#typical-energy-consumption-tec) can be used. However, SaaS and Managed Services are difficult to obtain relaible methodologies.
+
+{% include categoryLabel.html label="CatD" %}
+
+The downstream value chain; Category D (Downstream), considers the emissions of technology provided to the company's end users. This includes:
+- the electricity consumption of end user devices whilst using the company's public websites and APIs, software and SaaS services and products.
+- the fixed and radio transmission of data used by end users of those services.
+- the manufacture, transport, end-of-life use of produced technology hardware ([product carbon footprint](/resources/glossary#product-carbon-footprint-pcf)).
```

Comment:
```suggestion
- the manufacture, transport and end-of-life of produced technology hardware ([product carbon footprint](/resources/glossary#product-carbon-footprint-pcf)).
```
Perhaps "end-of-life processes" or similar if you want to clarify, but I think "end-of-life" is enough.